libraries/streaming-e2e-tests/bot/utils/renewDirectLineToken.js

```diff
@@ -0,0 +1,27 @@
+module.exports = async function (
+    token,
+    { domain = process.env.DIRECT_LINE_URL || 'https://directline.botframework.com/' } = {}
```

Comment:
Same as above? Should we always renew DL ASE type of token?

Reply:
ditto, no longer applicable (using DL ASE token gen endpoint, as your other comment pointed out)